src/pages/status.tsx

```diff
@@ -164,13 +164,32 @@ const Status = () => {
     router.push('/landing')
   }, [router])
 
+  function getTitleHeader(code: string | undefined): string {
```

Comment:
This does not account for "response-no-result" scenario. Difference between checkStatusResponse being undefined or null.

![image](https://github.com/user-attachments/assets/f8648608-f624-4952-9c51-c47e63146a89)

Reply:
Updated the cases to include null

Reply:
Updated version still isn't correct.

 switch (checkStatusResponse?.status) will evaluate to undefined in both null|undefined cases.
 
 Instead, early return on undefined.
 
if( checkStatusResponse === undefined) return t('header')

// else do the switch.

Reply:
Added in conditional to return for only undefined